content/applications/sales/point_of_sale/shop/serial_numbers.rst

```diff
@@ -0,0 +1,64 @@
+==============================
+Serial numbers and lots in PoS
+==============================
+
+Working with **serial numbers** and **lots** allows you to track the movements of your products.
+When products are tracked, the system can identify their location based on their last movement.
+
+To enable traceability, go to :menuselection:`Point of Sale --> Products --> Products`. Then,
+select a product and activate the :guilabel:`Tracking By Unique Serial Number` or the
```

Comment:
```suggestion
select a product and tick the :guilabel:`Tracking By Unique Serial Number` or the
```

Reply:
According to my research, "tick" is more used in British English :) However, I changed that sentence to "check the SN or LN box"